src/api/tests/service/creator_analytics/test_credit_detail.py

```
@@ -0,0 +1,477 @@
+"""End-to-end tests for POST /api/creator-analytics/credit-detail.
+
+The endpoint joins bill_usage × credit_ledger_entries server-side and
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)